resources/quality/rigid3d_base/nylon/rigid3d_base_nylon_good.inst.cfg

```diff
@@ -10,8 +10,8 @@ quality_type = good
 material = generic_nylon
 
 [values]
-default_material_print_temperature = 245
-default_material_bed_temperature = 90
+default_material_print_temperature = 250
+default_material_bed_temperature = 100
```

Comment:
Sorry! I missed this. 

We used to accept this in the past, but that's not correct. Basically you shouldn't set default_material_print_temperature or default_material_bed_temperature in quality profiles. The reason for this is that the materials (that are below the quality profiles) set this. If you want a certain material_print_temperature that is higher (so say that the default is set by the material to 240) you should use material_bed_temperature = =default_material_print_temperature + 10

Reply:
@nallath corrections are done as per your suggestion.